main.py

```diff
@@ -1 +1,22 @@
 #Mad Libs Generator Project
+loop = 1
+while (loop < 9):
```

Comment:
This code will loop 8 times. We need to loop 10 times. 
```suggestion
while (loop < 11):
```